proto/agent.proto

```diff
@@ -168,6 +168,13 @@ enum ContainerHealth {
 	unhealthy = 3;
 }
 
+// ContainerNetworkAddress records the IPs, Ports and Protocols for each container
+message ContainerNetworkAddress {
+	string ip = 1;
+	int32 port = 2;
+	string protocol = 3; // tcp/udp
```

Comment:
Should we change this to `ConnectionType` that we have defined below?

Reply:
Sure. I didn't see it.